var/spack/repos/builtin/packages/py-meshpy/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyMeshpy(PythonPackage):
+    """Simplicial Mesh Generation from Python"""
+
+    homepage = "https://documen.tician.de/meshpy/"
+    pypi = "MeshPy/MeshPy-2022.1.3.tar.gz"
+    git = "https://github.com/inducer/meshpy.git"
+
+    maintainers("cgcgcg")
+
+    version("2022.1.3", sha256="a7158e31ece25fa6c6cebce9fd1e968157d661dc8769fb30ceba69c351478475")
+
+    depends_on("python@3.6:", type=("build", "link", "run"))
+    depends_on("py-setuptools", type="build")
+    depends_on("py-pybind11", type="build")
+    depends_on("py-pytools@2011.2:", type=("build", "run"))
+    depends_on("py-numpy", type=("build", "link", "run"))
+    depends_on("py-gmsh-interop", type=("build", "run"))
+    depends_on("py-pytest@2:", type="build")
```

Comment:
pytest is only needed to run the unit tests, which this package doesn't. I would remove it